src/components/MultiSelect/MultiSelect.js

```diff
@@ -203,7 +203,7 @@ export default class MultiSelect extends React.Component {
                 <ListBox.Field {...getButtonProps({ disabled })}>
                   {selectedItem.length > 0 && (
                     <ListBox.Selection
-                      clearSelection={clearSelection}
+                      clearSelection={!disabled ? clearSelection : () => null}
```

Comment:
do we want to avoid creating a new instance of the anonymous function `() => null` each time a render is triggered?

Reply:
I'm not sure of a better solution, though I am sort of new to this. 👍

Reply:
```suggestion
                      clearSelection={!disabled ? clearSelection : noop}
```

Where `const noop = () => undefined` at the top level - Meaning, seconding @emyarod.